### PR TITLE
Add new reports

### PR DESCRIPTION
**Pull request by MarySammal**

### CODE REVIEW

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Open Reports
+author: Mary Sammal
+---
+# Open Reports
+
+You can use one of the following commands to open a report. The invoked **Save** dialog allows you to locate and open previously [saved reports](save-reports.md).
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md]: **Save** dialog? Not 'Open'?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+### Use the Ribbon Application Menu
+
+Click **Open...**.
+
+![eurd-win-ribbon-open-reports](../../../images/eurd-win-ribbon-open-reports.png)
+
+
+### Use the Ribbon Application Menu
+
+Click the application button and then **Open...** in the invoked application menu.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md]: "the application button"?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: Save Reports
+author: Mary Sammal
+---
+# Save Reports
+Create a report's _backup copy_ to guarantee that you can revert the report to its original state further.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md]: "Create a report's _backup copy_ to guarantee that you can revert the report to its original state further" What? This sentence does not make any sense. 'guarantee'? 'further'? Rewrite.

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Use the Ribbon's Home Tab
+
+* Click **Save** and the **Save as** to save your report's copy.
+	
+	![eurd-win-ribbon-save-report-as](../../../images/eurd-win-ribbon-save-report-as.png)
+	
+	Specify a folder and file name in the invoked standard **Save** dialog.
+
+* Click **Save** or press CTRL+S to save the report's current layout in the application's current folder.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md]: current... current?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+	
+	![eurd-win-ribbon-save-report](../../../images/eurd-win-ribbon-save-report.png)
+
+
+
+
+## Use the Ribbon Application Menu
+
+Click the application button. The invoked application menu provides the **Save** and **Save as** commands. These commands allow you to save the report's copy and the current layout, correspondingly.
```
Comment (dirk-pieterse) [inline interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md]: "the application button"? Which application button?

Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md]: "These commands allow you to save the report's copy and the current layout, correspondingly." One to save a copy, and the other to save the current layout?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md (hunk 1 above)
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md]: Yes, thanks, I'll change to Open.

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md (hunk 2 above)
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md]: This button is demonstrated on the image that follows this sentence.
image (switch to Preview)

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md (hunk 3 above)
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md]: I decided to simplify the information:

Use one of the following commands.

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md (hunk 4 above)
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md]: Click **Save** or press CTRL+S to save the report's layout in the application's current folder.

Ok?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md (hunk 5 above)
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md]: The application button is illustrated in the image below.

I reread. 
"The invoked application menu provides the **Save** and **Save as** commands. These commands allow you to save the current layout and the report's copy, correspondingly."
One to save the current layout and the other to save a copy. Ok?
Is the following better:
The invoked application menu provides the **Save** command to save the report's current layout, and the **Save as** command to save the report's copy.

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md (hunk 1 above)
Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md]: Ok

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md (hunk 2 above)
Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md]: This is what it is called? The 'application' button?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md (hunk 3 above)
Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md]: You can use one of the following commands to...:

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md (hunk 4 above)
Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md]: Click Save or press CTRL+S to save the report's layout in the application's folder.

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md (hunk 5 above)
Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/save-reports.md]: In the application menu, use the Save command to save the report's current layout, and the Save as command to save the report's copy.
OR
This menu provides the Save command to save the report's current layout, and the Save as command to save the report's copy.

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md (hunk 2 above)
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md]: Yes. https://documentation.devexpress.com/WindowsForms/8313/Controls-and-Libraries/Ribbon-Bars-and-Menu/Ribbon/Visual-Elements/Application-Button

**Pull request merged**